src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-remove-failed-node.ts

```diff
@@ -0,0 +1,240 @@
+                ${this.hass.localize(
+                  "ui.panel.config.zwave_js.remove_failed_node.remove_device"
+                )}
+              </mwc-button>
+            `
+          : ``}
+        ${this._status === "started"
+          ? html`
+              <div class="flex-container">
+                <ha-circular-progress active></ha-circular-progress>
+                <div class="status">
+                  <p>
+                    <b>
+                      ${this.hass.localize(
+                        "ui.panel.config.zwave_js.remove_failed_node.in_progress"
+                      )}
+                    </b>
+                  </p>
+                </div>
+              </div>
+            `
+          : ``}
+        ${this._status === "failed"
+          ? html`
+              <div class="flex-container">
+                <ha-svg-icon
+                  .path=${mdiCloseCircle}
+                  class="error"
+                ></ha-svg-icon>
+                <div class="status">
+                  <p>
+                    ${this.hass.localize(
+                      "ui.panel.config.zwave_js.remove_failed_node.removal_failed"
+                    )}
+                  </p>
+                  ${this._error
+                    ? html` <p><em> ${this._error.message} </em></p> `
+                    : ``}
+                </div>
+              </div>
+              <mwc-button slot="primaryAction" @click=${this.closeDialog}>
+                ${this.hass.localize("ui.panel.config.zwave_js.common.close")}
+              </mwc-button>
+            `
+          : ``}
+        ${this._status === "finished"
+          ? html`
+              <div class="flex-container">
+                <ha-svg-icon
+                  .path=${mdiCheckCircle}
+                  class="success"
+                ></ha-svg-icon>
+                <div class="status">
+                  <p>
+                    ${this.hass.localize(
+                      "ui.panel.config.zwave_js.remove_failed_node.removal_finished",
+                      "id",
+                      this._node!.node_id
+                    )}
+                  </p>
+                </div>
+              </div>
+              <mwc-button
+                slot="primaryAction"
+                @click=${this.closeDialogFinished}
+              >
+                ${this.hass.localize("ui.panel.config.zwave_js.common.close")}
+              </mwc-button>
+            `
+          : ``}
+      </ha-dialog>
+    `;
+  }
+
+  private async _startExclusion(): Promise<void> {
+    if (!this.hass) {
+      return;
+    }
+    this._status = "started";
+    try {
+      this._subscribed = await this.hass.connection.subscribeMessage(
+        (message) => this._handleMessage(message),
+        {
+          type: "zwave_js/remove_failed_node",
+          entry_id: this.entry_id,
+          node_id: this.node_id,
+        }
+      );
```

Comment:
Move to `data/zwave_js` and add typing to the message

Reply:
moved to data/zwave_js - the message could be a few things (error, or a couple different forwarded events from zwave-js with different info), so its just typed as `any` currently